src/util.jl

```diff
@@ -53,6 +53,68 @@ function audit(
     end
 end
 
+function update_names_count(
+        source::AbstractString,
+        name::AbstractString,
+        channel_id::UInt,
+        channel_name::AbstractString,
+        count_docs_dir = joinpath("data", "docs"),
+        count_names_file = joinpath(count_docs_dir, "$source.json")
+    )
+    
+    mkpath(count_docs_dir)
+    isfile(count_names_file) || write(count_names_file, "{}")
+    namescount = JSON.parsefile(count_names_file)
+
+    info = [
+        Dict(
+            "when" => "$(now(tz"UTC"))",
```

Comment:
I'm surprised that this can be parsed properly... I would otherwise surround that expression with triple-quotes.